### PR TITLE
Move remainder of ember tasks to client repo

### PR DESCRIPTION
**Pull request by acburdine**
refs #6977
- move ember build tasks to client repo
- remove unneeded dependencies

TODO:
- [x] update client ref once TryGhost/Ghost-Admin#177 is merged
- [x] ensure all commands work properly

### CODE REVIEW

**Comment by acburdine** [comment]
@ErisDS given that all of our grunt-shell commands are essentially proxies to npm commands - I was thinking of building a small wrapper around grunt-shell that essentially lets you run straight npm commands from grunt (maybe also an `npm-run` command that proxies to npm run <script-name> as well). End result is that you remove the unneeded shell config and do `grunt npm:shrinkwrap` instead of `grunt shell:shrinkwrap`.

File: Gruntfile.js [hunk 1]
```diff
@@ -59,7 +58,7 @@ var _              = require('lodash'),
                         'core/client/dist/index.html'
                     ],
                     options: {
-                        livereload: true
+                        livereload: 35729
```
Comment (acburdine) [inline Gruntfile.js]: Even though this number is the default livereload port - I thought it better to explicitly list it -> see the line comment in TryGhost/Ghost-Admin#177 for more details.

**Comment by acburdine** [comment]
Ready for review ⚡

**Comment by kirrg001** [comment]
`grunt test-client` says `Warning: Failed running "grunt test" in "core/client". Use --force to continue.`, i assume there is no grunt task für `test`, only `ember test` or `npm test`?

**Comment by acburdine** [comment]
Oh crap, there must not be - will fix soon.

**Comment by acburdine** [comment]
@kirrg001 fixed 😄

**Pull request merged**